src/loaders.py

```diff
@@ -149,6 +211,50 @@ def save(self, filepath):
         joblib.dump(params, filepath)
 
 
+class ImageSegmentationLoaderBasicTTA(BaseTransformer):
+    def __init__(self, loader_params, dataset_params):
+        super().__init__()
+        self.loader_params = AttrDict(loader_params)
+        self.dataset_params = AttrDict(dataset_params)
+
+        self.mask_transform = None
+        self.image_transform = None
+
+        self.image_augment = None
+        self.image_augment_with_target = None
+
+        self.dataset = ImageSegmentationTTADataset
```

Comment:
@taraspiotr if basic is an interface then self.dataset should be None imho

Reply:
@jakubczakon yes, of course. I changed it also in previous loader.